src/middlewares/json-api-express.ts

```diff
@@ -51,7 +51,7 @@ export default function jsonApiExpress(
     try {
       await authenticate(appInstance, req);
     } catch (error) {
-      res.status(error.status).json(convertErrorToHttpResponse(error));
+      res.status(+error.status).json(convertErrorToHttpResponse(error));
```

Comment:
@nelson6e65 I agree with this, I'm going to apply it with `Number` though for a more explicit and readable code:

```suggestion
      res.status(Number(error.status)).json(convertErrorToHttpResponse(error));
```

Reply:
You can use `Number.parseInt()` then, @joelalejandro